src/test/scala/tech/cryptonomic/conseil/tezos/michelson/parser/JsonParserSpec.scala

```diff
@@ -90,8 +90,8 @@ class JsonParserSpec extends FlatSpec with Matchers {
 
     parse[MichelsonInstruction](json) should equal(Right(
```

Comment:
If you prefer, scalatest will provide with combinators to extract `Right` using a DSL

```scala
parse[MichelsonInstruction](json).right should equal (...)
```